src/test/scala/com/cloudera/sparkts/models/ARIMAXSuite.scala

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2016, Cloudera, Inc. All Rights Reserved.
+ *
+ * Cloudera, Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"). You may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for
+ * the specific language governing permissions and limitations under the
+ * License.
+ */
+
+package com.cloudera.sparkts.models
+
+import breeze.linalg.{DenseMatrix, DenseVector => BreezeDenseVector}
+import org.apache.spark.mllib.linalg.DenseVector
+import org.scalatest.FunSuite
+import org.scalatest.Matchers._
+
+class ARIMAXSuite extends FunSuite {
+  // Data from http://www.robjhyndman.com/data/ - command to use this data available on website
+  // robjhyndman.com/talks/RevolutionR/exercises1.pdf
+  val gdp_train =  scala.io.Source.fromInputStream(getClass.getClassLoader.getResourceAsStream("data_train.csv"))
+    .getLines().drop(1).map(a => a.split(",",4).map(_.trim).slice(3,4).map(va => va.toDouble)).toArray.flatten
```

Comment:
It looks like there's a lot of repetition in these transformations.  Can they be factored into a helper method?